docs/apps.md

```diff
@@ -0,0 +1,505 @@
+is a better fit. We took a lot of inspirations from it, starting with the
+filename for this file: `manifest.webapp`.
+
+Field          | Description
+---------------|---------------------------------------------------------------------
+name           | the name to display on the home
+slug           | the default slug (it can be changed at install time)
+icon           | an icon for the home
+description    | a short description of the application
+source         | where the files of the app can be downloaded
+developer      | `name` and `url` for the developer
+default_locale | the locale used for the name and description fields
+locales        | translations of the name and description fields in other locales
+version        | the current version number
+license        | [the SPDX license identifier](https://spdx.org/licenses/)
+permissions    | a list of permissions needed by the app (see below for more details)
+contexts       | a list of contexts for the app (see below for more details)
+
+**TODO** [CSP policy](https://developer.mozilla.org/en-US/docs/Archive/Firefox_OS/Firefox_OS_apps/Building_apps_for_Firefox_OS/Manifest#csp)
+
+### Permissions
+
+An application has a list of permissions that the users has allowed. Each
+permission has a key, a description and an optional access level. The key is
+composed of two parts: the service that will perform the operation, and a
+subtype specific for each service. The description should explain why the
+permission is explained (the system can already gives a message on the "what"
+by using the key), and can be localized in the manifest.
+
+For data, the permission key is composed of `data/` and the doctype. The
+access can be `read`, `write` or `readwrite`.
+
+For files, the permission key is composed of `files/` and a type of files. The
+access can also be `read`, `write` or `readwrite`. The type can be :
+
+**Type**    | **Description**
+------------|---------------------------------------------------------
+`app`       | the folder `Apps/:app_name` and the files inside it
+`data`      | the folder `Documents/:app_name` and the files inside it
+`downloads` | the folder `Documents/downloads` and the file inside it
+`pictures`  | the folder `Documents/pictures` and the files inside it
+`music`     | the folder `Documents/music` and the files inside it
+`videos`    | the folder `Documents/videos` and the files inside it
+
+The `file/app` permission is powerful, it gives the app the possibility to
+modify itself. It can be dangerous, but it allows to create some static files
+for when JS is not an option. For example, a blog application can generate an
+RSS feed and upload it to this folder.
+
+For jobs, the permission key is composed of `jobs/` and the worker name. Some
+workers can use the `access` to restrict the permission (e.g. `konnectors` use
+the `access` to say which konnector can be used).
+
+For settings, the permission key is composed of `settings/` and a type. The
```

Comment:
Why separate settings instead of having `data/io.cozy.settings/locale` for instance ?